lib/express/morgan-middleware.js

```diff
@@ -13,6 +13,7 @@ module.exports = (getLogger, format) => {
     getLogger(req, res).child({
       http: line,
       route: res.locals.params || req.params,
+      traceparent: res.getHeader('x-request-id'),
```

Comment:
Although they requested for 'x-request-id', I think that we should follow the same used at our .net core apis:

- traceparent (which is at the recommendation)
- request-id (for compatibility reasons)

if we use x-request-id, they will have multiple implementations